x/campaign/types/key_mainnet_account.go

```diff
@@ -24,3 +24,19 @@ func MainnetAccountKey(campaignID uint64, address string) []byte {
 
 	return key
 }
+
+// MainnetAccountAllKey returns the store key to retrieve all MainnetAccount by campaign id
```

Comment:
I think we can try to simplify the keys methods like we do in `launch` accounts: https://github.com/tendermint/spn/blob/develop/x/launch/types/key_genesis_account.go

Reply:
https://github.com/tendermint/spn/pull/319/commits/76fbcffe1de6c2944f1d5a44eb12df86345c3558